doc/sources/tutorials/crashcourse.rst

```diff
@@ -0,0 +1,55 @@
+.. _crashcourse:
+
+Crash Course
+============
+
+Crash Course(or Kivy Crash Course) is a set of 14 video tutorials by Kivy core
+developer `inclement <https://github.com/inclement>`_ which provides a simple
+walkthrough in Kivy for users who know how to code in Python, but is also
+friendly even to beginners with Python as with Kivy. After Pong and Paint
+tutorials, this set of videos covers more features and tweaks that can be used
+to create your app quicker, your code elegant and eye-friendly.
+
+
+Basic Info
```

Comment:
I would agree about `Let's start`, but I'd say the other sections should stay. They should just be less specific - i.e. `Crash Course primarily consists of a series of YouTube videos.` (And YouTube should be written properly.) `Topics covered by the Crash Course include: ...`